### PR TITLE
Provisioning - password input screen rework

### PR DESCRIPTION
**Pull request by pzduniak**
@keybase/y2ksquad 
@keybase/react-hackers

### CODE REVIEW

File: shared/provision/password/index.tsx [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+          <Kb.LabeledInput
+            autoFocus={true}
+            placeholder="Password"
+            onEnterKeyDown={_onSubmit}
+            onChangeText={setPassword}
+            value={password}
+            textType="BodySemibold"
+            type="password"
+          />
+          <Kb.Text
```
Comment (buoyad) [inline shared/provision/password/index.tsx]: Impossible to tap this on a 5S-sized screen
image

File: shared/provision/password/container.tsx [hunk 2]
```diff
@@ -1,76 +1,21 @@
...
-      />
-    )
-  }
-}
+type OwnProps = {}
 
 const mapStateToProps = state => ({
-  error: state.provision.error.stringValue(),
-  waitingForResponse: WaitingConstants.anyWaiting(state, Constants.waitingKey),
+  error: state.provision.error.stringValue() || '',
```
Comment (buoyad) [inline shared/provision/password/container.tsx]: shouldn't need `|| ''`

**Comment by chrisnojima** [comment]
This should be fixed now. Not deferred

**Comment by buoyad** [comment]
@chrisnojima I mention that because the problem might be in `UserCard`, not exactly in scope of this ticket

**Comment by chrisnojima** [comment]
It doesn't have to be in this pr but it shouldn't be ticketed and deferred

**Pull request merged**